### ISSUE TITLE
Instabug (5.2.3) crashes on iOS7

### ISSUE THREAD

**Issue by zrxq** [Instabug/Instabug-iOS#68]
```
2016-05-12 12:48:27.698 ...[714:60b] -[UIScreen nativeScale]: unrecognized selector sent to instance 0x155a9170
(lldb)  po 0x155a9170
<UIScreen: 0x155a9170; bounds = {{0, 0}, {320, 480}}; mode = <UIScreenMode: 0x155a8ed0; size = 640.000000 x 960.000000>>

(lldb) bt
* thread #1: tid = 0xb07e, 0x38201ca8 libobjc.A.dylib`objc_exception_throw, queue = 'com.apple.main-thread', stop reason = breakpoint 35.2
   frame #0: 0x38201ca8 libobjc.A.dylib`objc_exception_throw
   frame #1: 0x2da54916 CoreFoundation`-[NSObject(NSObject) doesNotRecognizeSelector:] + 202
   frame #2: 0x2da53202 CoreFoundation`___forwarding___ + 706
   frame #3: 0x2d9a2768 CoreFoundation`_CF_forwarding_prep_0 + 24
   frame #4: 0x006776a8 ...`+[IBGInspector screenZoomed] + 120
   frame #5: 0x0067772a ...`+[IBGInspector getScreenScaleWithZoomMode] + 48
   frame #6: 0x00647578 ...`-[IBGDataStore initializeSnapshot] + 584
   frame #7: 0x386e9832 libdispatch.dylib`_dispatch_call_block_and_release + 10
   frame #8: 0x386e981e libdispatch.dylib`_dispatch_client_callout + 22
   frame #9: 0x386e9776 libdispatch.dylib`_dispatch_main_queue_callback_4CF$VARIANT$up + 254
   frame #10: 0x2da1b8a0 CoreFoundation`__CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 8
   frame #11: 0x2da1a174 CoreFoundation`__CFRunLoopRun + 1300
   frame #12: 0x2d984ebe CoreFoundation`CFRunLoopRunSpecific + 522
   frame #13: 0x2d984ca2 CoreFoundation`CFRunLoopRunInMode + 106
   frame #14: 0x3288a662 GraphicsServices`GSEventRunModal + 138
   frame #15: 0x302d114c UIKit`UIApplicationMain + 1136
 * frame #16: 0x0018b866 ...`main(argc=1, argv=0x27d9dc7c) + 106 at main.m:16
   frame #17: 0x3870eab6 libdyld.dylib`start + 2
```

**Comment by alliamin** [comment]
Hay @zrxq 
This issue is fixed in 5.2.4 Please update SDK.

**Comment by zrxq** [comment]
Thanks!
